x-pack/plugins/lens/public/datatable_visualization/components/table_basic.scss

```diff
@@ -1,3 +1,8 @@
+.lnsDataTableContainer {
+  height: 100%;
+  overflow: initial;
+}
+
 .lnsDataTable {
   align-self: flex-start;
 }
```

Comment:
Is this selector being used still? I didn't see it in my testing. If not, should probably be removed.